equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -36,8 +36,12 @@ public class CrmFileService {
     private static final String CRM_LINKED_EVIDENCE = "linkedAttachments";
     private static final String CRM_LINKED_EVIDENCE_FILES = "linkedAttachment";
     private static final String CRM14_CHARGES_BROUGHT = "Charges_brought";
-    private static final String CRM14_CHARGES = "row";
+    private static final String CRM14_ROW = "row";
```

Comment:
This should remain as CRM14_CHARGES. And if we need new constants, regardless of having the same value, we add them. That way, it is 100% clear what the mapping is being done and not just a simple "row"